test/helpers/mock.ts

```diff
@@ -90,14 +90,34 @@ export function withExternalServicesMock() {
 
     mockCarbon
       .intercept({
-        path: '/intensity/2023-12-01T00:00:00.000Z/2023-12-02T00:00:00.000Z',
+        path: '/intensity/2023-12-01T09:00:00.000Z/2023-12-02T10:00:00.000Z',
         method: 'GET',
       })
       .reply(200, {
         data: [
           {
-            from: '2023-12-01T00:00:00.000Z',
-            to: '2023-12-02T00:00:00.000Z',
+            from: '2023-12-01T09:00:00.000Z',
+            to: '2023-12-02T11:00:00.000Z',
+            intensity: {
+              actual: 123.456789123,
+              forecast: 100,
+              index: 'moderate',
+            },
+          },
+        ],
+      })
+      .persist()
+
+    mockCarbon
+      .intercept({
+        path: '/intensity/202023-01-01T06:00:00.000Z/2023-01-01T07:10:00.000Z',
+        method: 'GET',
+      })
+      .reply(200, {
+        data: [
+          {
+            from: '2023-01-01T06:00:00.000Z',
+            to: '2023-01-01T07:10:00.000Z',
```

Comment:
You've introduced an entirely new mock here but the test cases are basically the same. Is this an oversight maybe from when you had an additional integration test?

Reply:
Yes, indeed. again another leftover which has been removed as it's not needed (since it has been moved to the unit itself)